triage/labels.rst

```diff
@@ -10,405 +10,135 @@ Many labels are shared for both use cases, while some are dedicated
-| normal           | The default value for most issues filed.               |
-+------------------+--------------------------------------------------------+
-| deferred blocker | The issue will not hold up the next release, *n*. It   |
-|                  | will be promoted to a *release blocker* for the        |
-|                  | following release, *n+1*.                              |
-+------------------+--------------------------------------------------------+
-| release blocker  | The issue **must** be fixed before *any* release is    |
-|                  | made, e.g., will block the next release even if it is  |
-|                  | an alpha release.                                      |
-+------------------+--------------------------------------------------------+
-
-As a guideline, whether a bug is a *release blocker* for the current
-:ref:`release schedule <branchstatus>` is decided by the release manager.
-Triagers may recommend this priority and should notify the release manager by
-tagging them in a comment using ``@username``. If needed, consult the
-:ref:`release schedule <branchstatus>` and the release's associated PEP for the
-release manager's name.
-
-Keywords
---------
-Various informational flags about the issue. Multiple values are possible.
-
-+---------------+------------------------------------------------------------+
-|    Keyword    |                        Description                         |
-+===============+============================================================+
-| easy          | Fixing the issue should not take longer than a day for     |
-|               | someone new to contributing to Python to solve.            |
-+---------------+------------------------------------------------------------+
-
-Nosy List
----------
-A list of people who may be interested in an issue.
-
-This used to be a feature of the old issue tracker. On GitHub issues the
-same effect is achieved by tagging people in a comment using ``@username``.
-
-It is acceptable to tag someone to if you think the issue should be brought to
-their attention. Use the :ref:`experts` to know who wants to be added to the
-nosy list for issues targeting specific areas.
-
-If you want to subscribe yourself to an issue, click the *🔔 Subscribe*
-button in the sidebar. Similarly, if you were tagged by somebody else but
-decided this issue is not for you, you might click the *🔕 Unsubscribe*
-button in the sidebar.
-
-Assignees
----------
-Who is expected to take the next step in resolving the issue.
-
-It is acceptable to assign an issue to someone if the issue cannot move
-forward without their help, e.g., they need to make a technical decision to
-allow the issue to move forward. Also consult the :ref:`experts` as certain
-stdlib modules should always be assigned to a specific person.
-
-Note that in order to assign an issue to someone, that person **must** be
-a team member, likely a Triager or a core developer.
```

Comment:
This is briefly covered in the `issue-tracker.rst` page.

Reply:
It might still be helpful, though, to still provide some guidance about what the `Assignees` field should be used for, presumably on the `Triaging and Issue` page, which the section added to the Issue Tracker page references readers toward (but I can't find any more than a similarly brief mention of it either place). I can't find the details currently discussed here anywhere else, and this seems particularly germane given even among experienced triagers and core devs, there doesn't appear to be a lot of clarity or consistency regarding how to best use this field

Reply:
Seems like this is still missing; the mention on the `issue-tracker` page only states that regular users should not need to touch it and refers the reader to this page for more information, on which there now is none. I would suggest just restoring the text as-is for now and we can iterate on it in the future as necessary, as it is a net regression otherwise.

Reply:
I now moved in to `triaging.rst`.  I left a note about documenting the other fields too.

Reply:
Great, thanks!